From: Simon McVittie <smcv@debian.org>
Date: Mon, 3 May 2021 17:02:50 +0100
Subject: HACK: Disable Indic shaper

This appears to trigger a relayout loop in GTK 2 in the d-i environment.

Bug-Debian: https://bugs.debian.org/987587
---
 src/hb-ot-shape-complex.hh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hb-ot-shape-complex.hh b/src/hb-ot-shape-complex.hh
index a1a7a6a..6c82b87 100644
--- a/src/hb-ot-shape-complex.hh
+++ b/src/hb-ot-shape-complex.hh
@@ -263,7 +263,7 @@ hb_ot_shape_complex_categorize (const hb_ot_shape_planner_t *planner)
       else if ((planner->map.chosen_script[0] & 0x000000FF) == '3')
 	return &_hb_ot_complex_shaper_use;
       else
-	return &_hb_ot_complex_shaper_indic;
+	return &_hb_ot_complex_shaper_default;
 
     case HB_SCRIPT_KHMER:
 	return &_hb_ot_complex_shaper_khmer;
